### ISSUE TITLE
Can't restore data

### ISSUE THREAD

**Issue by niksabaldun** [jensstein/oandbackup#170]
Oab won't restore data of any app. I am on Sony Xperia M4 Aqua stock ROM (Marshmallow). The backup was made on the same ROM. Oab log and logcat output are attached. Logs were made with 0.2.10 debug version, but the same thing happens with 0.2.12 from F-Droid.

[oablogcat.txt](https://github.com/jensstein/oandbackup/files/1054219/oablogcat.txt)
[oablog.txt](https://github.com/jensstein/oandbackup/files/1054218/oablog.txt)

**Comment by ildar** [comment]
You seem to lack permission for reading external sd card. BTW the app
should toast about that, didn't you see?
That's another issue easily searchable.

**Comment by niksabaldun** [comment]
No, there was no toast. And I tried restoring from internal storage, with the same result.

**Comment by ildar** [comment]
Then would you provide the logcat from the moment of oab startup?

**Comment by niksabaldun** [comment]
Here is the complete logcat. I also noticed that oandbackup does not show apps which are not installed, even though they exist in backup folder. This happens when I put the backups on internal storage or external SD, regardless.

[oablogcat.txt](https://github.com/jensstein/oandbackup/files/1138373/oablogcat.txt)

**Comment by ildar** [comment]
> StrictMode policy violation; ~duration=6 ms:
android.os.StrictMode$StrictModeDiskReadViolation: policy=65567 violation=2

and lower.

It seems your Xposed denies oandbackup to Read Disk.

**Comment by niksabaldun** [comment]
I didn't even know I had Xposed. However, I installed Xposed Installer and turned Xposed off, but nothing changed with oab. Thanks for your help, but I give up. It seems oab is not actively developed any more, so it's obviously risky to use it.

**Comment by ildar** [comment]
I can be wrong with my analysis. Maybe others ( @jensstein ?) can do it
better?
